### DISCUSSION TITLE
Android 4.4-r5 w/ F-Droid and some goodies

### DISCUSSION THREAD

**Discussion by spetterman66** [copy/v86#1125] [Show and tell]
# Description
This disk image contains version 4.4-r5 of Android-x86, the latest version that works on v86 (until #1038 is implemented). It comes with F-Droid, SkyTube, an IRC client, and RedReader for reading Reddit. Due to requiring the ISRG Root X1 certificate for F-Droid to work, there is a PIN code present. The PIN is 1234.

# Download
Download it now at https://e.pcloud.link/publink/show?code=XZIc3gZlWgqkquUi9RLvUl945cUb7h6YedX

# Screenshots
![image](https://github.com/user-attachments/assets/8f59acc7-079f-4556-a123-cb6e4c6c0308)
PIN entry screen. There is an owner note added.
![image](https://github.com/user-attachments/assets/0561b550-6b1f-4dcb-97da-765a913d154f)
Home screen containing F-Droid, SkyTube, and RedReader
![image](https://github.com/user-attachments/assets/6b3b5b77-5b8d-47bd-92d4-1dffeebb071c)
Browsing r/food on RedReader
![image](https://github.com/user-attachments/assets/eaf5de4b-c36d-440f-9def-481c3a17fd34)
Updating repositories of the F-Droid app store
![image](https://github.com/user-attachments/assets/6e5ee48a-0d39-44d9-bf83-fd3c4b8e3e1c)
Also includes groundbreaking internet speed lmfao

### notes
The disk image is named `Other Linux 6.x kernel.img`, because that's the name of the VMware profile I have used to install this in

**Comment by Starkakrats** [comment]
Hello. How did you build android x86 image with preinstalled app?

**Reply by spetterman66** [reply]
you can just install it in qemu (or virtualbox, vmware, then convert file to img), then use that img file in v86
